library/centos

```diff
@@ -2,15 +2,6 @@ Maintainers: The CentOS Project <cloud-ops@centos.org> (@CentOS)
 GitRepo: https://github.com/CentOS/sig-cloud-instance-images.git
 Directory: docker
 
-Tags: latest, centos8, 8, centos8.4.2105, 8.4.2105
```

Comment:
Should the `latest` tag be moved to `centos:7` (as that will be the only supported version after this - until `stream8` and/or `stream9` are added that is)?

Reply:
Of course, that would be somewhat disruptive (but `:latest` by definition is "unpredictable", and not recommended, so perhaps that's "fine")

Reply:
I think removing it here means that it will still be there, just not updated (so continue to point to the `centos:8` image)